.github/workflows/lua-syntax.yml

```
@@ -16,9 +16,9 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - uses: leafo/gh-actions-lua@v8.0.0
+      - uses: leafo/gh-actions-lua@v10.0.0
```

```suggestion
      - uses: leafo/gh-actions-lua@v10
```

So it picks up latest v10 automatically (e.g. 10.1, 10.0.1)